forms-flow-web/src/components/AccessDenied/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { kcServiceInstance } from "../PrivateRoute"; // Import the kcServiceInstance function
+import accessDeniedIcon from "./AccessDenied.svg"; 
+import './accessDenied.scss';
+import { useTranslation } from "react-i18next";
+const AccessDenied = () => {
+  const { t } = useTranslation();
+  const handleLogout = () => {
+    const kcInstance = kcServiceInstance(); // Get the Keycloak instance
+    kcInstance.userLogout();
+  };
+
+  return (
+      <div className="d-flex flex-column  align-items-center text-center">
+      <img src={accessDeniedIcon} alt="Access Denied Icon" className="mb-4 mt-2"/>
+      <h1 className="access-denied-text">{t("Access Denied")}</h1>
+      <span className="access-denied">{t("You don't have permission to access this page.")}</span>

```

Comment:
Can we add these translations value mapped in translation files ?

Reply:
resolved